controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/UpdateReaderGroupTask.java

```diff
@@ -61,7 +61,7 @@ public UpdateReaderGroupTask(final StreamMetadataTasks streamMetaTasks,
         ImmutableSet<String> streamsToBeUnsubscribed = request.getRemoveStreams();
         final RGOperationContext context = streamMetadataStore.createRGContext(scope, readerGroup);
 
-        return RetryHelper.withRetriesAsync(() -> streamMetadataStore.getReaderGroupId(scope, readerGroup, context, executor)
```

Comment:
how is this change affecting the logic?

Reply:
this is not affect the table creation in any way, just a code improvement.